utils/collections.ts

```diff
@@ -0,0 +1,4 @@
+import type { Metadata } from '@/components/rmrk/service/scheme'
```

Comment:
Since we are dropping rmrk support, it would be better if we are not reusing from that

Reply:
just using the type that is already used inside `CollectionCard.vue`

![CleanShot 2024-04-25 at 15 02 41@2x](https://github.com/kodadot/nft-gallery/assets/44554284/f74e4ac0-496e-4095-81db-addd6c19487c)

can't find another type that matches this one

Reply:
> can't find another type that matches this one

https://github.com/kodadot/packages/blob/c1ccad8b6b6d42f642fa884784e19e4bd24155a9/hyperdata/src/types.ts#L7